react/Chips/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import MuiChip from '@material-ui/core/Chip'
+
+const Chips = ({ label, variant, color, ...props }) => {
+  return (
+    <MuiChip
+      className={cx({
+        noLabel: !label,
+        ghost: variant === 'ghost',
+        [`customColor-${color}`]: color
+      })}
+      label={label}
+      variant={variant === 'active' ? 'default' : 'outlined'}
+      color={variant === 'active' ? 'primary' : 'default'}
+      {...props}
+    />
+  )
+}
+Chips.displayName = 'Chips'
+
+Chips.propTypes = {
+  variant: PropTypes.oneOf(['default', 'active', 'ghost']),
+  color: PropTypes.oneOf(['primary', 'success', 'error', 'warning', 'info'])
```

Comment:
label is not a prop?
...props is not defined here?
don't we handle className?

Reply:
Exact pour label !

Pour les autres c'est ensuite des héritages Mui, je les mets pas je considère qu'on ensuite regarder la doc Mui...